### ISSUE TITLE
T-Rex miner not mining anymore

### ISSUE THREAD

**Issue by Jeypestylerz** [trexminer/T-Rex#513]
T-Rex is not mining anymore, i try to connect thought a VPN without results, I try to customize the command line and nothing else happens except this message:

![image](https://user-images.githubusercontent.com/86434125/123320660-566d0b00-d532-11eb-93ff-f651539935f3.png)

![image](https://user-images.githubusercontent.com/86434125/123321147-e57a2300-d532-11eb-8c6d-0c967ab98d27.png)

![image](https://user-images.githubusercontent.com/86434125/123321358-2e31dc00-d533-11eb-8f6b-eaa08ef33224.png)

I try to change the options, ports and IP directions, I prove that telnet was working but the http server don't;

![image](https://user-images.githubusercontent.com/86434125/123322064-1575f600-d534-11eb-94f4-c204d168d88e.png)

Can anyone check if in our HTTP direcction can see anything?.

I try any other mining by cmd software like Lolminer, Gminer, Xrig and in all of them I have the same problemm, I disabled the firewall also, no results.

With Xrig more problems:

![image](https://user-images.githubusercontent.com/86434125/123322354-79002380-d534-11eb-89e3-530b1ce03d4a.png)

It can be relationated with my cuda versión and OpenCL, I don´t know how to solve it.

Anyone has the same problem? Help me to investigate this problemm please I don't find any solution via internet.

**Comment by Jeypestylerz** [comment]
![image](https://user-images.githubusercontent.com/86434125/123322562-bebcec00-d534-11eb-851b-2c13f3f3c5a4.png)

With my loopback direction, the same..

**Comment by moms-spaghetti** [comment]
can you ping us-eth.2miners.com?

**Comment by Jeypestylerz** [comment]
@moms-spaghetti yes I can, I can ping any pool but any mining software that i try never works.

Before It has working I dont know what haved changed.

**Comment by MJZonni** [comment]
Same problem here, can u find solution?

**Comment by Leeeeerich** [comment]
I have the same problem
